doc/dev/installation.rst

```diff
@@ -87,6 +87,19 @@ They can be installed on Debian/Ubuntu via:
 
   sudo apt install clang lld ccache libomp-dev ninja-build make cmake
 
+They can be installed on macOS via:
+
+.. code-block:: console
+
+  brew install cmake ninja
+
+If you install Catalyst on a macOS system with ``ARM`` architecture (e.g. Apple M1/M2), you 
+additionally need to install the ``llvm-tools-preview`` rustup component:
+
+.. code-block:: console
+
+  rustup component add llvm-tools-preview
```

Comment:
Do we need to specify how to install rustup, or, just link to the website that shows how to install rustup?

Reply:
Hmm.. good point @josh146! I've added a link to the installation page of Rust (via rustup). This should be sufficient as installing Rust would also install rustup and cargo toolchains which are required for installing `llvm-tools-preview` and `qir-stdlib` from source.